stereo_image_proc/launch/stereo_image_proc.launch.py

```diff
@@ -215,7 +217,7 @@ def generate_launch_description():
         SetLaunchConfiguration(
             condition=LaunchConfigurationEquals('container', ''),
             name='container',
-            value='stereo_image_proc_container'
+            value='stereo_image_proc_container',
```

Comment:
Can we just write something like this (I didn't test it myself):
```suggestion
            value=[LaunchConfiguration('ros_namespace'), '/', 'stereo_image_proc_container']
```

Reply:
More or less except we need to wrap it in a `PythonExpression` to deal with the no-namespace case 
Fixed in latest commit https://github.com/ros-perception/image_pipeline/pull/752/commits/59d1677fa83a5fe954fe0a6fae0cad46966870f2